core/src/test/scala/com/broadinstitute/dsp/DbTestHelper.scala

```diff
@@ -42,12 +41,12 @@ object DbTestHelper {
   def insertDisk(disk: Disk, status: String = "Ready")(implicit xa: HikariTransactor[IO]): IO[Long] =
     insertDiskQuery(disk, status: String).withUniqueGeneratedKeys[Long]("id").transact(xa)
 
-  def insertK8sCluster(clusterId: KubernetesClusterId, status: String = "RUNNING")(implicit
+  def insertK8sCluster(cluster: KubernetesCluster, status: String = "RUNNING")(implicit
```

Comment:
Damn, this change from `KubernetesClusterId` to `KubernetesCluster` is super powerful. Is there an existing documentation you could point me to so I can read up on that?

Reply:
`KubernetesClusterId` comes from here in workbench-libs/google2: https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/kubernetesModels.scala#L95. It is really a GKE-specific thing.

`KubernetesCluster` is a model in the cronjobs repo, and supports both clouds.

The naming is so similar, and confused me for a bit. 😅 No real documentation except staring at code.

….And maybe worth chatting about later, I feel like the Leo data model for k8s cluster/nodepool/app in general. The code is kind of confusing and the data model is hard to work with. But not high priority, maybe future tech debt.